astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -139,9 +142,11 @@ def execute_complete(
             elif "status" in event and event["status"] == "success":
                 hook = self.get_db_hook()
                 qids = typing.cast(List[str], event["query_ids"])
-                hook.check_query_output(qids)
+                handler = self.handler or fetch_all_handler
+                results = hook.check_query_output(qids, handler)
                 self.log.info("%s completed successfully.", self.task_id)
-                return None
+                if self.do_xcom_push:
+                    return results
         else:
             self.log.info("%s completed successfully.", self.task_id)
             return None
```

Comment:
We can remove this line, right? By default a function return `None`, no?

Reply:
in case of no event from the trigger, I think should be an error case, WDYT?

Reply:
Yeah if no event is passed we need to raise an error, now fixed it